src/js/components/Banner.js

```diff
@@ -9,25 +9,29 @@ export default class Banner extends React.PureComponent {
     title: '',
     message: '',
     toggleBannerVisibilty: false,
-    year: 0,
-    day: 0,
-    month: 0,
+    date: 0,
+    mysqlDate: 0,
+    index: 0,
   };
 
   componentDidMount() {
     const d = new Date();
-    const $yr = d.getFullYear();
-    const $month = d.getMonth();
-    const $day = d.getDate();
-    this.setState({ year: $yr, day: $day, month: $month })
-    fetch(`http://api.sikhitothemax.org/messages/web/${$yr}-${$month}-${$day}`)
+    const $mysqlDate = d.getFullYear() + '-' + (d.getMonth() + 1) + '-' + d.getDate() + ' ' + d.getHours() + ':' + d.getMinutes() + ":" + d.getSeconds();
+    const $date = $mysqlDate.slice(0, 10);
+
+    this.setState({ mysqlDate: $mysqlDate, date: $date });
+    fetch(`//api.sikhitothemax.org/messages/web/${$date}`)
```

Comment:
Use webpack.config.js for API URLs instead of writing it directly in code.

Take a look at how we are using other API links in code.